pipeline/mutate/mutator_hydrator.go

```diff
@@ -184,26 +188,31 @@ func (a *MutatorHydrator) Mutate(r *http.Request, session *authn.AuthenticationS
 	}
 	req.Header.Set(contentTypeHeaderKey, contentTypeJSONHeaderValue)
 
-	var client http.Client
+	var client *retryablehttp.Client
 	if cfg.Api.Retry != nil {
 		maxRetryDelay := time.Second
-		giveUpAfter := time.Millisecond * 50
+		// giveUpAfter := time.Millisecond * 50 this was previously used as maxelapsedtime for the old transport implementation
```

Comment:
@aeneasr : Can you please help with this one?

Reply:
You mean whether this default is ok?